tests/validator_test.py

```diff
@@ -14,3 +19,14 @@ def test_validator_result(X: List[str], y: List[Union[str, int]], expected_resul
     validator = Validator(create_data_fixture(X, y))
     validator.validate()
     assert validator.result == expected_result
+
+
+@pytest.mark.parametrize('X, y, expected_result', [
+    (['', '', ''], [1, 2, 1], 'failed'),
```

Comment:
can you also add a 'passed' case. incase implementation change result in always 'failed'